pkg/scd/operations_handler.go

```diff
@@ -223,6 +224,20 @@ func (a *Server) PutOperationReference(ctx context.Context, req *scdpb.PutOperat
 		State:          scdmodels.OperationState(params.State),
 	}, key)
 
+	if err == scderr.MissingOVNsInternalError() {
+		// The client is missing some OVNs; provide the pointers to the
+		// information they need
+		ops, err := a.Store.SearchOperations(ctx, uExtent, owner)
+		if err != nil {
+			return nil, err
+		}
+		err, success := scderr.MissingOVNsErrorResponse(ops)
+		if !success {
+			return nil, dsserr.Internal(fmt.Sprintf("failed to construct missing OVNs error message: %s", err))
```

Comment:
do you mean to use this error in the Sprintf?

Reply:
Yes; that indicates why we failed to construct the error we wanted to construct.